### PR TITLE
Cleanup and update peopleops page

### PR DESCRIPTION
**Pull request by cas--**
The new onboarding template layout requires new steps for peopleops so
update the document and clean it up.

Refs: niteoweb/operations#387

### CODE REVIEW

File: roles/people-ops.md [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+# People operations role ([PeopleOps])
 
-## Checklist for trialist onboarding
+**Contents**
+  1. [Onboarding](#onboarding)
+      1. [Trialist](#trialist)
+      1. [Permanent](#permanent-position)
+  3. [Offboarding](#offboarding)
 
-A member of the [PeopleOps] team will do the following to start the [onboarding](onboarding.md) process:
```
Comment (karantan) [inline roles/people-ops.md]: I would not remove this.

File: roles/people-ops.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+## Onboarding
 
-* Send the <a href="mailto:?
+> *Reference document: [Onboarding](onboarding.md)*
+
+
+### Trialist
+
+When a new team member is hired on trial basis, a member of the [PeopleOps] team should do the following to start the [onboarding](onboarding.md) process:
```
Comment (karantan) [inline roles/people-ops.md]: `team should do the following`: should? We need to start the onboarding process. It is not an option.

File: roles/people-ops.md [hunk 3]
```diff
@@ -39,7 +51,12 @@ I've also created your onboarding checklist so you can continue with the onboard
 %0D%0A
 Regards%0D%0A%0D%0A
 "> account credentials </a> email.
-* Once email encryption is setup, send the <a href="mailto:@niteo.co?
+
+
+#### Trialist checklists
+
+1. Create a new [trialist welcome issue] and assign it to the new team member.
+    * Once email encryption is set up, send the <a href="mailto:@niteo.co?
```
Comment (karantan) [inline roles/people-ops.md]: I would throw encrypted email out and add signal. It is a lot easier to setup and we already decided that we will start using it.

Comment (zupo) [reply roles/people-ops.md]: That will be done as part of https://github.com/niteoweb/operations/issues/448.

File: roles/people-ops.md (hunk 2 above)
Comment (cas--) [reply roles/people-ops.md]: You have raised an interesting point about language dynamics. I didn't feel `will` was correct here and yes `should` in the strictest sense does imply an option but often used as a 'soft way' of say it must be done. 

I did find a useful article on the subject that could be part of the handbook guidelines: http://reqexperts.com/blog/2012/10/using-the-correct-terms-shall-will-should/

Based on that I would agree with the article that `shall` is a much better alternative.

**Pull request merged**